content/chronograf/v1.9/about_the_project/release-notes-changelog.md

```diff
@@ -8,6 +8,43 @@ menu:
     parent: About the project
 ---
 
+## v1.9.4 [2022-03-28]
+
+### Features
+
+This release renames the Flux `Query Builder` to the Flux `Script Builder` (and adds improvements), and improves on Kapacitor integration.
+
+#### Flux Builder improvements
+
+- Rename the Flux `Query Builder` to the Flux `Script Builder`, and add new functionality including:
+  - Ability to load truncated data into the Flux Script Builder.
```

Comment:
Ability to load truncated tags/fields into the Flux Script Builder when connected to InfluxDB Cloud.

Reply:
Done